src/modules/change-state.js

```diff
@@ -247,7 +247,9 @@
 			var isMaxWidthValid = (!!maxWidth && window.mediaQueryMaxWidth(maxWidth)) || !maxWidth;
```

Comment:
I've talked about this problem with @PascalPiche and the problem boils down to reads: we either need to do them ALL on the first element (old behaviour) or ALL on EACH element. 

Since the behaviour we want is for ALL element to EACH have their own state, we must (at the bare minimum) move the 4 reads (lines 244-247) into the new loop.

Reply:
fixer dans rd2645e3